CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 0.1.4 (XX-XX-XXXX)
 ------------------
 
+ - Update language selector templatetag to check for pages without languages
```

Comment:
Maybe you can add a more clear description here?

``Update language selector templatetag to work with pages without language as well``